base/.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+// Place your settings in this file to overwrite default and user settings.
```

Comment:
maybe a good addition to contrib in a separate PR, if vscode can be made to look at settings files in different locations?